android/src/main/java/expo/modules/geofencingbatchplugin/WoosmapExpoLifecycleListener.kt

```diff
@@ -0,0 +1,53 @@
+package expo.modules.geofencingbatchplugin
+
+import android.app.Application
+import android.content.Context
+import android.content.IntentFilter
+import android.os.Build
+import android.util.Log
+import com.batch.android.Batch
+import com.batch.android.BatchActivityLifecycleHelper
+import expo.modules.core.interfaces.ApplicationLifecycleListener
+
+class WoosmapExpoLifecycleListener : ApplicationLifecycleListener {
+
+    companion object {
+        private const val TAG = "WoosmapExpoLifecycle"
+        private const val BATCH_API_KEY = "BATCH_API_KEY"
+    }
+
+    override fun onCreate(application: Application) {
+        super.onCreate(application)
+        startBatch(application)
```

Comment:
`startBatch(application)` should not be part of our plugin; this responsibility lies with the batch SDK.